tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/newstyle:2019_04_05/exception_javascript_no_in_app.pysnap

```diff
@@ -0,0 +1,46 @@
+---
+created: '2019-04-09T18:49:36.326246Z'
+creator: sentry
+source: tests/sentry/grouping/test_variants.py
+---
+app:
+  hash: None
+  component:
+    app (exception of system takes precedence)
+      exception (ignored because this variant does not have a contributing stacktrace, but the system variant does)
```

Comment:
This snapshot shows the improvement. Before we produced a hash of just `TypeError` which is obviously too broad.